NSonic/Impl/Connections/IngestConnection.cs

```diff
@@ -163,7 +163,7 @@ public PushRequest(string text, string locale)
                 this.locale = locale;
             }
 
-            public string Text => $"\"{this.text}\"";
+            public string Text => $"\"{this.text.Replace("\n", "\\n").Replace("\"", "\\\"")}\"";
```

Comment:
The `\` itself also needs a escape and should be done by first:
```suggestion
            public string Text => $"\"{this.text.Replace("\\", "\\\\").Replace("\n", "\\n").Replace("\"", "\\\"")}\"";
```